avm/utilities/pipelines/platform/Set-AvmGithubIssueForWorkflow.ps1

```diff
@@ -0,0 +1,128 @@
+<#
+.SYNOPSIS
+Check for failing pipelines and create issues for those, that are failing.
+
+.DESCRIPTION
+If a pipeline fails, a new issue will be created, with a link to the failed pipeline. If the issue is already existing, a comment will be added, if a new run failed (with the link for the new failed run). If a pipeline run succeeds and an issue is open for the failed run, it will be closed (and a link to the successful run is added to the issue).
+
+.PARAMETER Repo
+Mandatory. The name of the respository to scan. Needs to have the structure "<owner>/<repositioryName>"
+
+.PARAMETER LimitNumberOfRuns
+Optional. Number of recent runs to scan for failed runs. Default is 100.
+
+.PARAMETER LimitInDays
+Optional. Only runs in the past selected days will be analyzed. Default is 2.
+
+.PARAMETER IgnoreWorkflows
+Optional. List of workflow names that should be ignored (even if they fail, no ticket will be created). Default is an empty array.
+
+.EXAMPLE
+Set-AvmGithubIssueForWorkflow -Repo 'owner/repo01' -LimitNumberOfRuns 100 -LimitInDays 2 -IgnoreWorkflows @('Pipeline 01')
+
+Check the last 100 workflow runs in the repository 'owner/repo01' that happened in the last 2 days. If the workflow name is 'Pipeline 01', then ignore the workflow run.
+
+.NOTES
+The function requires GitHub CLI to be installed.
+#>
+function Set-AvmGithubIssueForWorkflow {
+  [CmdletBinding(SupportsShouldProcess, ConfirmImpact = 'Low')]
```

Comment:
Again, why the `ConfirmImpact`? There must be a reason you added it last time and this time again :D please don't keep me in the dark. I'm not saying you should remove it, but would like to understand the reason :)

Reply:
The reason is simple: copy & paste :-D . I will remove it